music21/musicxml/xmlToM21.py

```diff
@@ -6206,9 +6144,9 @@ def testStaffLayoutMore(self):
         self.assertEqual(sl0.topDistance, 211.0)
         self.assertEqual(sl0.leftMargin, 70.0)
         self.assertEqual(sl0.rightMargin, 0.0)
-#         for s in layouts:
-#             if hasattr(s, 'staffSize'):
-#                 print(s, s.staffSize)
+        #         for s in layouts:
```

Comment:
Comments such as these were designed so that the comment could be deleted and the code would run.  Now if the comment is deleted we get an indentation error.